index.html

```diff
@@ -165,23 +165,96 @@ <h1>What is a Hackathon?</h1>
         <h1>Schedule</h1>
         <p>Times are listed in your local time zone.</p>
         <div id="schedule-block">
-            <div id="schedule-tapes">
-                <div class="tape"></div>
-                <div class="tape"></div>
+
+            <!-- Binder rings -->
+            <div id="binder-container">
+                <div class="binder-rings">
+                    <div class="binder-ring"></div>
+                    <div class="binder-ring"></div>
+                </div>
+                <div class="binder-rings">
+                    <div class="binder-ring"></div>
+                    <div class="binder-ring"></div>
+                </div>
             </div>
-            <div id="schedule-content">
-                <div id="days">
-                    <div class="day day-first day-active">
-                        <h2 class="number">20</h2>
-                        <h6 class="name">SATURDAY</h6>
+
+            <div id="schedule-tabs">
+                <div class="schedule-tab pre-event schedule-tab-active">
+                    <p>Pre-Event</p>
+                </div>
+                <div class="schedule-tab feb-20">
+                    <p>Feb 20th</p>
+                </div>
+                <div class="schedule-tab feb-21">
+                    <p>Feb 21st</p>
+                </div>
+            </div>
+
+            <div id="schedule-content-wrapper">
+                <div id="pre-event-content" class="schedule-content">
+                    <div class="day-row pre-16">
+                        <div class="day">
+                            <h2 class="number">16</h2>
+                            <h6 class="name">TUESDAY</h6>
+                        </div>
+                        <div class="events pre-event-16-events"></div>
+                    </div>
+                    <div class="day-row pre-17">
+                        <div class="day">
+                            <h2 class="number">17</h2>
+                            <h6 class="name">WEDNESDAY</h6>
+                        </div>
+                        <div class="events pre-event-17-events"></div>
+                    </div>
+                    <div class="day-row pre-18">
+                        <div class="day">
+                            <h2 class="number">18</h2>
+                            <h6 class="name">THURSDAY</h6>
+                        </div>
+                        <div class="events pre-event-18-events"></div>
+                    </div>
+                    <div class="day-row pre-19">
+                        <div class="day">
+                            <h2 class="number">19</h2>
+                            <h6 class="name">FRIDAY</h6>
+                        </div>
+                        <div class="events pre-event-19-events"></div>
+                    </div>
+                </div>
+                <div id="feb-20-content">
+                    <div class="schedule-content">
+                        <div class="days">
```

Comment:
Really confused by the structure here. Consider making it match pre-event's more, especially if you need to do something like move the date above the events.

Reply:
https://github.com/codeRIT/brickhack.io/issues/1161